tests/integration/helpers.py

```diff
@@ -885,3 +886,134 @@ async def get_cluster_status(ops_test: OpsTest, unit: Unit) -> Dict:
     get_cluster_status_action = await unit.run_action("get-cluster-status")
     cluster_status_results = await get_cluster_status_action.wait()
     return cluster_status_results.results
+
+
+async def delete_file_or_directory_in_unit(ops_test: OpsTest, unit_name: str, path: str) -> bool:
+    """Delete a file in the provided unit.
+
+    Args:
+        ops_test: The ops test framework
+        unit_name: The name unit on which to delete the file from
+        path: The path of file or directory to delete
+
+    Returns:
+        boolean indicating success
+    """
+    try:
+        return_code, _, _ = await ops_test.juju("ssh", unit_name, "sudo", "rm", "-rf", path)
```

Comment:
ah, it would be terrible to have `rm -rf` run by mistake. changed to using `find` in [a0052bf](https://github.com/canonical/mysql-operator/pull/329/commits/a0052bfc237344ffb5aa266203468dedc5d41d53) (with `maxdepth=0` to match exact paths, and no `mtime` as the files the test deletes will be recently created)